Dev

kareraolivier opened the pull request:
# PR Description

This Pull Request contain the development of ojemba single page website, the technologies It is built in are html, tailwind and javascript. It was built according to the figma design we have on this link [https://www.figma.com/file/YRF8ZXuCSDqwmBI9iurvVC/Ojemba.de-VLM-(Copy)?type=design&node-id=0%3A1&mode=design&t=rPH3Qzjc4b42DYhe-1 ](url).
We have also provided the link of the hosted finished version of ojemba website here [https://t9-ojemba-clone.netlify.app/](url)


## Description of tasks that were expected to be completed

- [x] Creating a configured html and tailwind project
- [x]  Build the website 
- [x]  Build the responsiveness to all devices big and small one


## Description of tasks that were expected to be completed and that are not yet completed

- [x] none


## How this may be tested?

1. run `git clone` [https://github.com/devfabien/T9-Ojemba-clone.git](url) after getting the project
2. run `git checkout dev` to get to the branch containing the full website
3. run `npm i` to install all required dependencies
4. run `npm run build` to build tailwind styles
5. run `npm run start` to start the project
6. Open live server if you have it or Open index file manually

## Here are the Screenshots 

![Screenshot Capture - 2023-08-30 - 20-28-46](https://github.com/devfabien/T9-Ojemba-clone/assets/77101006/e8085376-8ab0-4b6d-9aef-f876c60369ac)

![Screenshot Capture - 2023-08-30 - 20-30-03](https://github.com/devfabien/T9-Ojemba-clone/assets/77101006/c7662f92-5000-4932-8843-1fa8f0b49a39)

![Screenshot Capture - 2023-08-30 - 20-32-38](https://github.com/devfabien/T9-Ojemba-clone/assets/77101006/49b39635-8e31-4565-ab09-5b935f3bf086)


![Screenshot Capture - 2023-08-30 - 20-30-22](https://github.com/devfabien/T9-Ojemba-clone/assets/77101006/f5466cbd-f1bd-4aca-9484-b26a1c2295a7)

![Screenshot Capture - 2023-08-30 - 20-33-43](https://github.com/devfabien/T9-Ojemba-clone/assets/77101006/d998c9b1-3271-4c47-8957-ab72809ad8a6)

![Screenshot Capture - 2023-08-30 - 20-34-20](https://github.com/devfabien/T9-Ojemba-clone/assets/77101006/05b82f31-baa2-466d-be4d-b9a3704bebd2)

![Screenshot Capture - 2023-08-30 - 20-34-53](https://github.com/devfabien/T9-Ojemba-clone/assets/77101006/a4febd81-2c54-43b3-ae31-868f46f2a829)

![Screenshot Capture - 2023-08-30 - 20-44-18](https://github.com/devfabien/T9-Ojemba-clone/assets/77101006/6a84917a-613f-436b-8f7a-6c915bffc096)

![Screenshot Capture - 2023-08-30 - 20-45-34](https://github.com/devfabien/T9-Ojemba-clone/assets/77101006/bf17af5b-a4ee-41a6-ac17-b4870e7a5e61)


## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

devfabien committed:
chore(setting-project):
-tailwind configuration
[Finish #001]

devfabien committed:
chore(setting-project):
-tailwind configuration
[Finish #001]

kunda4 committed:
chore(PR-template): create Pull Request Template
- Add a github folder and PR file
[Finish #002]

kareraolivier committed:
Merge pull request #1 from devfabien/pull-request-template

Ch-#002: create Pull Request Template

kareraolivier committed:
#003 : Collecting and adding  all assets

kunda4 committed:
Merge pull request #2 from devfabien/ch/asset

Ch-#003 : Collecting and adding  all assets

kunda4 committed:
(ch-tailwind): Configure Tailwing

-Setting all colors we will use in taiwind.config.js
-Setting fonts we will use in tailwind.config.js

[Finish #004]

kareraolivier committed:
Merge pull request #3 from devfabien/ch-configure-tailwind

Ch-#004: Configure Tailwind

kunda4 committed:
(ft-footer): Hero Section Footer

-Create a single card and mapping throuth it.
-Export data for using it somewhere else

[Finish #007]

devfabien committed:
Feat: Adding Managed team
adding data components
mapping through accordions data

devfabien committed:
Feat: Adding Managed team
adding data components
mapping through accordions data

devfabien committed:
Merge pull request #4 from devfabien/ft-#007-hero-footer

Ft -#007: Hero Section Footer

devfabien committed:
Feat: creating managed page part2

devfabien committed:
fix:resolving conflicts

devfabien committed:
fix: changing the image dimensions

kunda4 committed:
Merge pull request #5 from devfabien/ft/#008managedTeam

Feat: #008Adding Managed team

kunda4 committed:
Merge pull request #5 from devfabien/ft/#008managedTeam

Feat: #008Adding Managed team

kunda4 committed:
(ft-#010): Work on Unsere Team Page

-Create content for the page(Title&Paragrapgh)
-Add video
-style Page

[Finish #010]

kareraolivier committed:
navbar

devfabien committed:
(ft-Skill): Skills set page

- Working #011: Skill set page
- Pair Programing in #012: Skill set page part2

[Finish #011, #012]

kareraolivier committed:
ft-#017 : contact and footer page
This contain accordions and contact button
this PR also contain footer
[#017 : Fragen und Antworten Page]

kareraolivier committed:
ft-#006 : Creating Hero section
This PR contain hero section
this PR also contain index file configuration
[#006 : Creating Hero section]

kunda4 committed:
(ft-Maiborn): working on MainbornWolf page

- working on required style needed to the content

[Finish #014]

kareraolivier committed:
Merge branch 'dev' of https://github.com/devfabien/Team-Project into dev

kunda4 committed:
Merge pull request #8 from devfabien/ft/navba

Ft/navbar and hero section

kareraolivier committed:
Merge branch 'dev' of https://github.com/devfabien/Team-Project into dev

devfabien committed:
Managed TeamPage #009
adding the rest accordions from part1
adding the maibornwolf logo

kareraolivier committed:
Merge branch 'dev' into ft/Fragen-und-Antworten-Page

devfabien committed:
Merge branch 'dev' into ft/#009Mnagedteampage

kareraolivier committed:
Merge pull request #9 from devfabien/ft/#009Mnagedteampage

Managed TeamPage #009

kareraolivier committed:
Merge branch 'dev' of https://github.com/devfabien/Team-Project into dev

kunda4 committed:
Merge branch 'dev' into ft-unsere-team

kareraolivier committed:
Merge branch 'dev' into ft/Fragen-und-Antworten-Page

kunda4 committed:
Merge pull request #7 from devfabien/ft/Fragen-und-Antworten-Page

ft-#017 : contact and footer page

kunda4 committed:
(ft-page): work on training programming page

- Add contents on the page
- Style the page
- work on responsviness of the page

[Finish #013]

kunda4 committed:
Merge branch 'dev' into ft-unsere-team

kareraolivier committed:
Merge pull request #10 from devfabien/ft-unsere-team

ft-#010: Unsere Team Page

kareraolivier committed:
Merge pull request #11 from devfabien/ft-#013-training

ft-#013: Training Program Page

kareraolivier committed:
ft/#016 : Ruanda – ein Land der Sub-Sahara mit viel Potential Page
this PR contain map and content
Also contain founder page
Also map page is responsive

kareraolivier committed:
Merge branch 'dev' into ft-#015-founder-page

kareraolivier committed:
Merge branch 'dev' into ft-MaibornWolff-page

kareraolivier committed:
Merge branch 'dev' into ft-MaibornWolff-page

kareraolivier committed:
Merge branch 'ft-MaibornWolff-page' of https://github.com/devfabien/T9-Ojemba-clone into ft-MaibornWolff-page

kareraolivier committed:
Merge pull request #12 from devfabien/ft-MaibornWolff-page

ft-#014: MaibornWolff x The Gym x Maliyo Page

kareraolivier committed:
Merge branch 'dev' into ft-#015-founder-page

devfabien committed:
Merge branch 'dev' into ft/skillSet

devfabien committed:
Merge branch 'dev' into ft/skillSet

kareraolivier committed:
Merge pull request #6 from devfabien/ft/skillSet

ft-#012: skills set Pages

kareraolivier committed:
Merge branch 'dev' into ft-#015-founder-page

devfabien committed:
Merge pull request #13 from devfabien/ft-#015-founder-page

ft/#016 : Ruanda – ein Land der Sub-Sahara mit viel Potential Page

kareraolivier committed:
fix-#019:whole-website-preview-heropage-css
this card contain basic website style
fix of javascript cards imports
[finish #019]

kunda4 committed:
Merge pull request #14 from devfabien/Fx/heropage-css

fix-#019:whole-website-preview-heropage-css

devfabien committed:
(fx-responsive): fixing the the responsiveness of all project.

- Fix the responsiveness on mobile phone
- Fix the responsiveness on tablet

 [Finish #020]

kareraolivier committed:
Merge pull request #15 from devfabien/fx/#020-responsiveness

(fix-responsive): fixing the the responsiveness of all project.

kareraolivier committed:
#030: Fixing the footer
background have been changed to the right one
button text color is changed
accordion paddding is added
[#fix footer is done]

kareraolivier committed:
#030: Fixing the footer
background have been changed to the right one
button text color is changed
accordion paddding is added
[#fix footer is done]

devfabien committed:
Fixing the card #024, #026, #027
#024 : fixing managed team page
#026 : fixing the Qualitat section
#027 : fixing founder page

kunda4 committed:
(fx-QA): working on quality assuraance of the project

- fix navbar by adding padding and work on font
- fix skills page by change image size and content size

[Finish #025]

kunda4 committed:
(fx-QA): working on quality assuraance of the project

- fix navbar by adding padding and work on font
- fix skills page by change image size and content size

[Finish #025]

kareraolivier committed:
Merge branches 'Fx/footerpage' and 'Fx/footerpage' of https://github.com/devfabien/T9-Ojemba-clone into fx/footerpage

kunda4 committed:
Merge pull request #16 from devfabien/Fx/footerpage

#030: Fixing the footer

devfabien committed:
Fixing the requested changes:
adding the font size

devfabien committed:
Merge branch 'dev' into fix/QAfeedback

kunda4 committed:
Merge pull request #18 from devfabien/fix/QAfeedback

Fixing the card from #024: fixing managed team page

kunda4 committed:
Merge branch 'dev' into fx-quality-assurance

kunda4 committed:
(fx-QA): working on Quality Assurance of the project

kunda4 committed:
Merge branches 'fx-quality-assurance' and 'fx-quality-assurance' of https://github.com/devfabien/Team-Project into fx-quality-assurance

kareraolivier committed:
Merge pull request #17 from devfabien/fx-quality-assurance

(fx-#025): working on quality assurance of the project

kunda4 committed:
(fx-conflict): resolve conflict

devfabien committed:
Merge pull request #19 from devfabien/fx-quality-assurance2

(fx-#30): resolve conflict

devfabien committed:
Fixing QA feedback

adjusting the spacing
updating the content

kareraolivier committed:
Merge pull request #20 from devfabien/fix/coach-QA

Fixing QA feedback #032

devfabien committed:
Fixing underline

adding green svg
adjusting the max width

devfabien committed:
Merge branch 'dev' into fx/underlines

devfabien committed:
Merge branch 'dev' into fx/underlines

kunda4 committed:
Merge pull request #21 from devfabien/fx/underlines

Fx/underlines

devfabien committed:
Merge branch 'dev' into fx/underlines

kareraolivier committed:
Merge pull request #22 from devfabien/fx/underlines

Fx/underlines

IbrahimBagalwa commented on .gitignore:
```diff
@@ -103,7 +106,7 @@ dist
 .temp
 .cache
-
+dist/
```
I noticed that the `dist/` directory is listed multiple times. While this might be intentional, make sure you're not accidentally duplicating entries.

IbrahimBagalwa commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Is not a good practice to push this `.vscode/settings.json` to a public repository on GitHub.

IbrahimBagalwa commented on index.html:
```diff
@@ -1,11 +1,66 @@
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400;1,500&display=swap" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400;1,500&display=swap"
+        rel="stylesheet">
```
It is a good practice to download fonts and host or add them directly to your assets folder.

IbrahimBagalwa commented on index.html:
```diff
@@ -1,11 +1,66 @@
+
+        </div>
+        <div class="bg-gradient-to-r from-backgroundColor from-40% to-primary/50" style="-webkit-clip-path: polygon(0 0, 100% 0, 100% 100%, 0 96%);
+        clip-path: polygon(0 0, 100% 0, 100% 100%, 0 96%);">
```
`style="-webkit-clip-path: polygon(0 0, 100% 0, 100% 100%, 0 96%);
        clip-path: polygon(0 0, 100% 0, 100% 100%, 0 96%);` instead of using clip-path it might be good if you use `skew` because `-webkit-clip-path` can make your code harder to maintain, especially when you need to update or modify the styles in the future.

IbrahimBagalwa commented on package.json:
```diff
@@ -1,9 +1,10 @@
+    "start": "tailwindcss -i ./src/input.css -o ./dist/output.css --watch"
+  },
+  "dependencies": {
+    "tailwind-scrollbar-hide": "^1.1.7",
```
Do you really need this package ? do not add features that is not included in your current task

IbrahimBagalwa commented on src/assets/fonts/Montserrat/OFL.txt:
```diff
@@ -0,0 +1,93 @@
+distributed by the Copyright Holder(s).
+
+"Modified Version" refers to any derivative made by adding to, deleting,
+or substituting -- in part or in whole -- any of the components of the
```
Do you really need this file?

IbrahimBagalwa commented on src/assets/fonts/Montserrat/README.txt:
```diff
@@ -0,0 +1,81 @@
+  static/Montserrat-Black.ttf
+  static/Montserrat-ThinItalic.ttf
+  static/Montserrat-ExtraLightItalic.ttf
+  static/Montserrat-LightItalic.ttf
```
Here as well

IbrahimBagalwa commented on src/assets/fonts/Raleway/OFL.txt:
```diff
@@ -0,0 +1,93 @@
+requirement for fonts to remain under this license does not apply
+to any document created using the fonts or their derivatives.
+
+DEFINITIONS
```
Why this file?

IbrahimBagalwa commented on src/assets/fonts/Raleway/README.txt:
```diff
@@ -0,0 +1,81 @@
+  static/Raleway-BoldItalic.ttf
+  static/Raleway-ExtraBoldItalic.ttf
+  static/Raleway-BlackItalic.ttf
+
```
Here as well

IbrahimBagalwa commented on src/components/data.js:
```diff
@@ -0,0 +1,143 @@
+        `<div class="rounded-2xl py-1 px-2 h-fit bg-white whitespace-nowrap text-backgroundColor font-semibold font-title text-sm">${skill} </div>`
+    )
+    .join("")}`;
+};
```
To make this code more readable and maintainable,  it might be good to break your code into smaller functions that are responsible for specific tasks.

IbrahimBagalwa commented on src/components/data.js:
```diff
@@ -0,0 +1,143 @@
+      description: "+",
+    },
+  ];
+  let acc4 = document.getElementById("accordion4");
+  acc4.innerHTML = `${accordionone
```
maintain a consistent naming convention for your variables and functions. eg: `managedTeam`

kareraolivier replied:
done

IbrahimBagalwa commented on src/components/data.js:
```diff
@@ -0,0 +1,143 @@
+    )
+    .join("")}`;
+  let acc1 = document.getElementById("accordion1");
+  console.log("jdfdhjf", accordion);
```
No need to print this here, it would be good if you remove it.

kareraolivier replied:
It is removed

IbrahimBagalwa commented on src/components/data.js:
```diff
@@ -0,0 +1,143 @@
+        ` <div class=" flex justify-between font-bold font-para px-2 pt-12 items-bottom border-b-2 border-primary w-full"><h2>${data.title}</h2> <h2>${data.description}</h2></div>`
+    )
+    .join("")}`;
+  let acc1 = document.getElementById("accordion1");
```
Here as well find a meaningful name

kareraolivier replied:
It is fixed with skillcontainer

IbrahimBagalwa commented on src/components/data.js:
```diff
@@ -0,0 +1,143 @@
+    },
+    {
+      title: "100% Verantwortung",
+      description: "not ready",
+    },
+    {
+      title: "Riesiger Talentepool ",
+      description: "not ready",
+    },
+    {
+      title: "Risikoarm ",
+      description: "not ready",
+    },
+  ];
+  const accordionone = [
+    {
+      title: "Welche Frage steht hier?",
+      description: "+",
+    },
+    {
+      title: "Welche Frage steht hier?",
+      description: "+",
+    },
+    {
+      title: "Welche Frage steht hier? ",
+      description: "+",
+    },
+    {
+      title: "Welche Frage steht hier?",
+      description: "+",
+    },
+    {
+      title: "Welche Frage steht hier? ",
+      description: "+",
+    },
+    {
+      title: "Welche Frage steht hier? ",
+      description: "+",
+    },
+  ];
```
If possible, separate data fetching from content generation.

kareraolivier replied:
We did separate them

IbrahimBagalwa commented on src/components/data.js:
```diff
@@ -0,0 +1,143 @@
+const managedTeam = () => {
+  const accordion = [
```
Your `accordionone` and `accordion` arrays seem to be similar but used separately. Consider using a single array and filtering it as needed to reduce redundancy.

kareraolivier replied:
solved and done

IbrahimBagalwa commented on src/components/data.js:
```diff
@@ -0,0 +1,143 @@
+        `<div class="rounded-2xl py-1 px-2 h-fit bg-white whitespace-nowrap text-backgroundColor font-semibold font-title text-sm">${skill} </div>`
+    )
+    .join("")}`;
+};
```
Keep consistent formatting and indentation to improve readability.

kareraolivier replied:
yes

IbrahimBagalwa commented on src/components/data.js:
```diff
@@ -0,0 +1,143 @@
+    .join("")}`;
+
+  let acc2 = document.getElementById("accordion2");
+  console.log("jdfdhjf", accordion);
```
Here as well remove this console

kareraolivier replied:
It is Removed

IbrahimBagalwa commented on src/input.css:
```diff
@@ -1,3 +1,19 @@
+@font-face {
+    font-family: 'Roboto';
+    src: url('./assets/fonts/Roboto/Roboto-Medium.ttf');
+}
```
Keep your formatting consistent. For example, you've used single quotes for `'Roboto'` but double quotes for the other font families. Pick one style and stick to it for consistency.

IbrahimBagalwa commented on src/sections/MaibornWolff.html:
```diff
@@ -0,0 +1,43 @@
+
+    <body class="bg-backgroundColor">
+        <div class=" bg-white w-full px-8 lg:px-32 pb-32 " style="-webkit-clip-path: polygon(0 0, 100% 11%, 100% 100%, 0 89%);
+        clip-path: polygon(0 0, 100% 11%, 100% 100%, 0 89%);">
```
Here as well remove the `-webkit-clip-path`

IbrahimBagalwa commented on src/sections/MaibornWolff.html:
```diff
@@ -0,0 +1,43 @@
+
+
+    <body class="bg-backgroundColor">
```
Having multiple <body> tags in the same HTML document is not valid HTML and can lead to rendering issues and unexpected behavior.

IbrahimBagalwa commented on src/sections/MaibornWolff.html:
```diff
@@ -0,0 +1,43 @@
+                        <div class="flex gap-4">
+                            <img class="w-8" src="./src/assets/image 1.svg" alt="">
+                            <img class="w-8 md:w-20" src="./src/assets/sthl-preview.webp" alt="">
+                            <img class="w-8 md:w-20" src="./src/assets/Miele_Logo_M_Red_sRGB.jpg" alt="">
```
In order to improve accessibility for screen readers and users with disabilities `alt` attributes for images, should have a meaningful

IbrahimBagalwa commented on src/sections/MaibornWolff.html:
```diff
@@ -0,0 +1,43 @@
+                        <div class="flex gap-4">
+                            <img class="w-12 md:w-20" src="./src/assets/Google_2015_logo 1.jpg" alt="">
+                            <img class="w-12 md:w-32" src="./src/assets/31823-img-au_logo 1.jpg" alt="">
+                        </div>
```
You're using `w-16`, `w-32`, etc., for image sizing. Consider using responsive classes like `md:w-1/2` for images to adjust their size based on the screen width.

IbrahimBagalwa commented on src/sections/MaibornWolff.html:
```diff
@@ -0,0 +1,43 @@
+                <h1 class="font-title text-lg font-bold">MaibornWolff x The Gym x Maliyo</h1>
+                <p class=" text-sm pb-10">Für das Training werden die Erfahrung und das Wissen über erfolgreiche
+                    Softwareprojekte kombiniert mit dem standortspezifischen ...</p>
+            </div>
```
Keep your formatting consistent

IbrahimBagalwa commented on src/sections/founder.html:
```diff
@@ -0,0 +1,129 @@
+
+                <div class="z-20"><img src="./src/assets/group.webp" alt="people" class=""></div>
+                <div style="-webkit-clip-path: polygon(53% 7%, 72% 31%, 100% 34%, 100% 100%, 0 84%, 0 34%, 34% 29%); clip-path: polygon(53% 7%, 72% 31%, 100% 34%, 100% 100%, 0 84%, 0 34%, 34% 29%); "
+                    class="bg-backgroundColor/95 bg-gradient-to-l from-darkPrimary/30 w-full text-white lg:pt-40 py-16 lg:px-40 px-4 absolute mx-auto -bottom-3 lg:-bottom-28 md:-bottom-10 left-0 right-0 z-10">
```
Keep your formatting consistent

IbrahimBagalwa commented on src/sections/founder.html:
```diff
@@ -0,0 +1,129 @@
+
+    <body class="bg-gradient-to-l ">
```
Here as well change this body tag

IbrahimBagalwa commented on src/sections/founder.html:
```diff
@@ -0,0 +1,129 @@
+                        und Süden anbieten.
+
+                    </p>
+                    <div></div>
```
If no content you can just remove this div

IbrahimBagalwa commented on src/sections/founder.html:
```diff
@@ -0,0 +1,129 @@
+                                <stop offset="1" stop-color="#8ACC50" stop-opacity="0.06"/>
+                                </linearGradient>
+                                </defs>
+                                </svg>
```
It could be good if you move this svg in assets in order to make your code readable

IbrahimBagalwa commented on src/sections/founder.html:
```diff
@@ -0,0 +1,129 @@
+                                </svg>
+                                
+                        </span> Markt und Talentpool</h1>
+                        <ul class="">
```
No need to add a empty `class`

IbrahimBagalwa commented on src/sections/founder.html:
```diff
@@ -0,0 +1,129 @@
+            </div>
+            <div class="flex justify-center items-center ">
+                <img src="/src/assets/Africa.webp" alt="" srcset=""
+                    class="absolute lg:w-1/3 md:w-6/12 lg:right-40 md:flex hidden right-10">
```
Make sure to provide descriptive and meaningful `alt` text that conveys the purpose of the image.

IbrahimBagalwa commented on src/sections/founder.html:
```diff
@@ -0,0 +1,129 @@
+                        takimata sanctus est L
+
+                    </p>
+                    <div></div>
```
here as well if there is no content just remove it

IbrahimBagalwa commented on src/sections/footer.html:
```diff
@@ -0,0 +1,120 @@
+<body>
+    <div style="background: rgb(138,204,80);
+    background: linear-gradient(140deg, rgba(138,204,80,1) -70%, rgba(0,19,66,1) 29%, rgba(0,19,66,1) 64%, rgba(138,204,80,1) 110%);"
```
It's generally a good practice to separate your CSS styles from your HTML code. Consider moving your inline styles to external stylesheets or using classes.

IbrahimBagalwa commented on src/sections/footer.html:
```diff
@@ -0,0 +1,120 @@
+                    <path
+                        d="M72.0195 28.4609C71.7639 28.6361 71.473 28.7636 71.1635 28.8361C70.854 28.9085 70.5318 28.9246 70.2155 28.8833C69.8992 28.842 69.5949 28.7442 69.32 28.5954C69.0451 28.4466 68.8049 28.2498 68.6132 28.0162C67.1467 26.2146 65.2379 24.7529 63.0404 23.7487C60.8429 22.7445 58.4181 22.2258 55.9613 22.2346C55.4828 22.2345 55.015 22.1055 54.6163 21.8638C54.2177 21.622 53.9058 21 ...
+                        fill="white" />
+                </svg>
```
Ensure that your SVGs are accessible by providing appropriate `aria-label` or `title` attributes. This will help screen readers convey meaningful information about the SVG content.

IbrahimBagalwa commented on src/sections/footer.html:
```diff
@@ -0,0 +1,120 @@
+                                <path
+                                    d="M64.2452 5.35045C64.2452 2.39548 61.8742 0 58.9495 0C56.0248 0 53.6538 2.39548 53.6538 5.35045C53.6538 8.30542 56.0248 10.7009 58.9495 10.7009C61.8742 10.7009 64.2452 8.30542 64.2452 5.35045Z"
+                                    fill="#8ACC50" />
+                            </svg></a></span>
```
Move the svgs into assets to make your code more readable

IbrahimBagalwa commented on src/sections/footer.html:
```diff
@@ -0,0 +1,120 @@
+                    </div>
+                </div>
+            </div>
+        </div>
```
Stick consistency by formatting your code

IbrahimBagalwa commented on src/sections/hero.html:
```diff
@@ -0,0 +1,76 @@
+        </nav>
+        <div>
+
+        </div>
```
Avoiding Unnecessary Elements

IbrahimBagalwa commented on src/sections/hero.html:
```diff
@@ -0,0 +1,76 @@
+                <h1 class="text-textColorwhite text-2xl lg:text-5xl align-start font-semibold font-title">Win-Win:<br />
+                    Europas <span class="text-primary">Digitalisierung</span>
+                    <br />profitiert von Afrikas Talenten
+                </h1>
```
Don’t use `br` instead of using `br` use `p` or `span`

IbrahimBagalwa commented on src/sections/hero.html:
```diff
@@ -0,0 +1,76 @@
+                                d="M1.7348 37.2935C0.576889 34.1486 0 30.5677 0 26.7645C0 22.9613 0.576889 19.3762 1.7348 16.2355C5.35378 6.28942 14.1854 0.00390625 25.9803 0.00390625C37.7752 0.00390625 46.6069 6.29361 50.2259 16.2355C51.3838 19.3804 51.9648 22.9655 51.9648 26.7645C51.9648 30.5635 51.3838 34.1486 50.2259 37.2935C46.6816 47.1641 37.9205 53.5293 26.055 53.5293C14.1896 53.5293 5.353 ...
+                                fill="#8ACC50" />
+                            <path
+                                d="M48.9478 59.2281H50.6825C53.7246 59.2281 54.7373 57.6934 54.7373 54.6198V16.8145H63.2784V54.6911C63.2784 62.44 59.2984 66.4654 51.4088 66.4654H48.9478V59.2239V59.2281Z"
```
When using SVG icons, include an accessible aria-label attribute for screen readers to describe the icon's purpose. For example:
```html
<span class="lg:hidden cursor-pointer" aria-label="Toggle Menu">
    <!-- Your SVG -->
</span>

IbrahimBagalwa commented on src/sections/heroFooter.html:
```diff
@@ -0,0 +1,48 @@
+                <h5 class="font-para text-primary font-bold flex items-center"><span >Kostengünstig</h5>
+                <p class="font-para md:block hidden text-textColorwhite">niedriges Lohnkostenniveau
+                    im europäischen Vergleich</p>
+                </div>
```
what do you mean by `text-textColorwhite`

IbrahimBagalwa commented on src/sections/heroFooter.html:
```diff
@@ -0,0 +1,48 @@
+                    <defs> <clipPath id="clip0_405_90">  <rect width="107" height="107" fill="white" /> </clipPath> </defs>
+                </svg>
+                <div class="text-sm">
+                <h5 class="font-para text-primary font-bold flex items-center"><span >Kostengünstig</h5>
```
Is this `<span>` here necessary?

IbrahimBagalwa commented on src/sections/managedTeam.html:
```diff
@@ -0,0 +1,78 @@
+        </div>
+    </div>
+  
+</body>
```
it might be good if you format this code for readability.

IbrahimBagalwa commented on src/sections/managedTeam.html:
```diff
@@ -0,0 +1,78 @@
+<body class="bg-backgroundColor">
+    <div class="relative bg-secondary
+     py-20 md:px-28 px-6" style="-webkit-clip-path: polygon(0 0, 100% 0, 100% 100%, 0 95%);
+    clip-path: polygon(0 0, 100% 0, 100% 100%, 0 95%);">
```
It could be good if you use another alternative than `-webkit-clip-path` for the maintainability of the code

IbrahimBagalwa commented on src/sections/skillSet.html:
```diff
@@ -0,0 +1,38 @@
+                    </svg>killset un</span>serer
+            Entwickler:innen</h2>
+        <div class="py-1 flex gap-3 font-medium font-title text-sm overflow-y-auto" id="skills"></div>
+        <div class="py-1 flex gap-3 font-medium font-title text-sm overflow-y-auto" id="skills2"></div>
```
Your attributes should have a meaningful

IbrahimBagalwa commented on src/sections/skillSet.html:
```diff
@@ -0,0 +1,38 @@
+                    </svg>amkonstellation</span>
+        </h2>
+        <div class=" flex flex-col justify-center items-center md:py-1 relative ">
+            <img src="./src/assets/teamAnol.webp" alt="" srcset="" class="md:max-w-3xl max-w-xs">
```
Here as well the `alt` attribute should have a meaningful for the accessibility

IbrahimBagalwa commented on src/sections/trainingProgram.html:
```diff
@@ -0,0 +1,50 @@
+                Lead IT Architect bei MaibornWolff,
+                der das Ausbildungsprogramm maßgeblich entwickelt hat:</h2>
+
+            <!-- <div class="flex justify-center lg:pb-52 py-10"><img class="" src="./src/assets/Video 2.webp" alt=""> -->
```
If you no longer need this code just remove it

IbrahimBagalwa commented on src/sections/unsereTeam.html:
```diff
@@ -0,0 +1,37 @@
+            </div>
+            <div class="flex justify-center">
+                <!-- <img class="md:px-20 px-4 md:py-10" src="./src/assets/Video 1.webp" alt=""> -->
+                <!-- <video src="https://youtu.be/DhzKg3FJFDw" class="w-full h-96"></video> -->
```
Here as well please remove unnecessary code instead of commenting.

IbrahimBagalwa commented on src/sections/unsereTeam.html:
```diff
@@ -0,0 +1,37 @@
+            <div class="flex justify-center">
+                <!-- <img class="md:px-20 px-4 md:py-10" src="./src/assets/Video 1.webp" alt=""> -->
+                <!-- <video src="https://youtu.be/DhzKg3FJFDw" class="w-full h-96"></video> -->
+                <iframe class="w-full md:px-28 md:h-[700px] px-4 h-[300px]" src="https://www.youtube.com/embed/DhzKg3FJFDw" title="Working at MaibornWolff" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Avoid mixing px and rem units. Go for one

IbrahimBagalwa commented on tailwind.config.js:
```diff
@@ -1,8 +1,23 @@
+  theme: {
+    extend: {
+      colors: {
+        darkPrimary: "#116355",
+        primary: "#8ACC50",
+        secondary: "#F1F4F6",
+        textColorblack: "#000000",
+        textColorwhite: "#FFFFFF",
+        backgroundColor: "#001342",
+        yellow: "#F4BA00",
```
Most of the colors exist in tailwind and don't really know why do you need this here.

IbrahimBagalwa commented on tailwind.config.js:
```diff
@@ -1,8 +1,23 @@
+        backgroundColor: "#001342",
+        yellow: "#F4BA00",
+      },
+      fontFamily: {
+        poppins: ["Poppins", "sans-serif"],
+        para: ["Montserrat", "sans-serif"],
+        title: ["Raleway", "sans-serif"],
+      },
```
For specific width, height, and spacing. Please, follow the tailwind way used to name them according to the content, not their usage.

IbrahimBagalwa commented on tailwind.config.js:
```diff
@@ -1,8 +1,23 @@
-  }
\ No newline at end of file
+  },
+  plugins: [require("tailwind-scrollbar-hide")],
```
Remove the feature that is not included in your task.

IbrahimBagalwa reviewed (changes requested):
Great job team 9 👍  
Your HTML code looks well-structured, but ensure you consider accessibility practices and use semantic HTML elements.
If you already have this webp images extension no need to have PNG or jpg
Avoid putting space between tailwind classes eg:  classeName="py-10   w-fit" 
Try to customize more your README with more information related to your specific usage of your project.

kunda4 committed:
(ch-readme): Create ReadMe file and work QA

- Create ReadMe file of our project
- remove unwanted dependecies in package.json
- work on code review QA

[Finish #34 #39]

kunda4 committed:
(ch-readme): Create ReadMe file and work QA

- Create ReadMe file of our project
- remove unwanted dependecies in package.json
- work on code review QA

[Finish #34 #39]

kunda4 committed:
Merge branch 'ch-readme' of https://github.com/devfabien/Team-Project into ch-readme

devfabien committed:
Fixing code reviews

configuration of tailwind
adding external css
removing unnecessary files

devfabien committed:
fixing svgs

keeping consistency
importing svgs

kunda4 committed:
Merge pull request #25 from devfabien/fx/images_tailwindconfig

Fx/images, tailwindconfig and consistency

kunda4 committed:
Merge remote-tracking branch 'origin' into ch-readme

kunda4 committed:
Merge remote-tracking branch 'origin' into ch-readme

kareraolivier committed:
#38: fix and restructure javascript files
renaming variables in more readable way
crating new file for data
aranging my functions and restructuring the code
[#fix restructure js files]

kareraolivier committed:
Merge branch 'dev' into fix-and-restructure-javascript-files

kareraolivier committed:
Merge branch 'dev' into fix-and-restructure-javascript-files

kareraolivier committed:
Merge branch 'fix-and-restructure-javascript-files' of https://github.com/devfabien/T9-Ojemba-clone into fix-and-restructure-javascript-files

devfabien committed:
Merge pull request #26 from devfabien/fix-and-restructure-javascript-files

Fix and restructure javascript files

kunda4 committed:
Merge remote-tracking branch 'refs/remotes/origin/ch-readme' into ch-readme

kunda4 committed:
Merge remote-tracking branch 'origin' into ch-readme

kunda4 committed:
fix gitignore

kunda4 committed:
fix gitignore

devfabien committed:
Merge pull request #24 from devfabien/ch-readme

ch-readme: Create ReadMe file and work QA

IbrahimBagalwa reviewed (approved):
LGTM 👍
